docs/essentials/introduction.md

```diff
@@ -11,6 +11,28 @@ A major strength of Storybook are [addons](/addons/) that extend Storybook’s U
 - [Backgrounds](./backgrounds.md)
 - [Toolbars & globals](./toolbars-and-globals.md)
 
+### Installation
```

Comment:
Probably should note that this comes pre-installed with `sb init`.

Reply:
@maslade can you address the feedback that @shilman mentioned? Once that's addressed i think that we'll be good to go. 

Let me know once you've taken care of it.

Reply:
You bet - updated!